src/components/editMeasure/populationCriteria/groups/QDM/QDMMeasureGroups.tsx

```diff
@@ -453,6 +456,56 @@ const MeasureGroups = (props: MeasureGroupProps) => {
     }
   };
 
+  const getReturnTypes = (
+    populations: Array<Population | MeasureObservation>
+  ) => {
+    return populations?.reduce((returnTypes, population) => {
+      const definition = _.camelCase(_.trim(population?.definition));
+      const returnType = cqlDefinitionDataTypes[definition];
+      if (returnType) {
+        returnTypes.push(returnType);
+      }
+      return returnTypes;
+    }, []);
+  };
+  const getReturnTypesStrats = (populations: Array<Stratification>) => {
+    return populations?.reduce((returnTypes, population) => {
+      const definition = _.camelCase(_.trim(population?.cqlDefinition));
```

Comment:
I wish it was `definition` instead of `cqlDefinition`

Reply:
@adongare Yeah i was thinking the same!!